cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -103,6 +103,56 @@ module cosmos 'br/public:avm/res/document-db/database-account:0.4.0' = {
   }
 }
 {{- end}}
+{{- if .DbCosmos }}
+
+module cosmos 'br/public:avm/res/document-db/database-account:0.8.1' = {
```

Comment:
We can merge new `cosmos` with existing `cosmos`. But if `.Cosmos` and `.DbCosmosMongo` not appear at the same time, the generated bicep are the same, keep 2 `cosmos`s will make it easier to understand.